src/components/SideMenu/SideMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import './SideMenu.styles.scss';
+import ChurchIcon from '@mui/icons-material/Church';
+import SettingsIcon from '@mui/icons-material/Settings';
+import LogoutIcon from '@mui/icons-material/Logout';
+import Tab from './Tab';
```

Comment:
I think we should configure `eslint/sort-import` or something like, to make npm imports first, then components (tsx) then css/utils last.

Reply:
done